src/clusterfuzz/_internal/bot/tasks/task_creation.py

```diff
@@ -76,7 +76,7 @@ def create_blame_task_if_needed(testcase):
     return
 
   # Blame is only applicable to chromium project, otherwise bail out.
-  if testcase.project_name != 'chromium':
+  if testcase.is_chromium():
```

Comment:
I was trying to understand more how predator integration works and I noticed this.

I think this needs to be `if not testcase.is_chromium()`

This may have turned off predator for chromium completely?

Reply:
https://github.com/google/clusterfuzz/pull/3569